frontend/src/components/messages/AssistanceRequestTable.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <v-data-table-virtual
+    v-if="assistanceRequests?.length > 0"
+    v-model="bodyCheckboxesSelected"
+    :headers="headers"
+    :items="assistanceRequests"
+    item-key="assistanceRequestId"
+    item-value="assistanceRequestId"
+    height="65vh"
+    show-select
+    hover
+    fixed-header
+    class="tableText"
+    density="compact"
+    @click:row="rowClickHandler">
+    <template #item.status="{ item }">
+      <div :class="getItemClass(item)">
+        {{ item.status }}
+      </div>
+    </template>
+    <template #item.subject="{ item }">
+      <div :class="getItemClass(item)">
+        {{ item.subject }}
+      </div>
+    </template>
+    <template #item.categoryName="{ item }">
+      <div :class="getItemClass(item)">
+        {{ item.categoryName }}
+      </div>
+    </template>
+    <template #item.lastConversationTime="{ item }">
+      <div :class="getItemClass(item)">
+        {{ convertDateToLocalDate(item.lastConversationTime) }}
+      </div>
+    </template>
+  </v-data-table-virtual>
+</template>
+
+<script>
+import { mapState, mapActions } from 'pinia'
+import { useMessagesStore } from '@/stores/messages'
+
+export default {
+  name: 'AssistanceRequestTable',
```

Comment:
Not a big deal but try to follow a standard order for components:

https://eccbc.atlassian.net/wiki/spaces/OOFMCC/pages/152698987/OFMCC+-+Portal+Home

Reply:
when I followed the orders in the Confluence page, I got the Eslints warnings (i.e.: The "props" property should be above the "data" property and The "watch" property should be above the "methods" property).

Reply:
Ok. We can revisit our standards and/or linter config.